sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Generated/Models/CapabilityBase.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Models
+{
+    /// <summary> Base object for representing capability. </summary>
+    public partial class CapabilityBase
+    {
+        /// <summary> Initializes a new instance of CapabilityBase. </summary>
+        internal CapabilityBase()
+        {
+        }
+
+        /// <summary> Initializes a new instance of CapabilityBase. </summary>
+        /// <param name="status"> The status of the capability. </param>
+        /// <param name="reason"> The reason for the capability not being available. </param>
+        internal CapabilityBase(PostgreSqlFlexbileServerCapabilityStatus? status, string reason)
+        {
+            Status = status;
+            Reason = reason;
+        }
+
+        /// <summary> The status of the capability. </summary>
+        public PostgreSqlFlexbileServerCapabilityStatus? Status { get; }
```

Comment:
I see why the ApiCompat is reporting the missing Status - we move it to the new base type, this is not breaking, but we changed its type, this is breaking.

This is how we resolve issues like this: 
1. rename this property to something else like `CapabilityStatus` to make room for the name `Status`. (using rename-mapping)
2. add the `string Status` back using customization code, we could have this: `public string Status => CapabilityStatus.ToString();`
and then this should be fixed.